pkg/backup/backup.go

```diff
@@ -53,9 +53,21 @@ type BackupController struct {
 	recentBackupsStatus []backupapi.BackupStatus
 }
 
+// BackupControllerConfig contains configuration data to construct BackupController.
+type BackupControllerConfig struct {
+	Kclient kubernetes.Interface
```

Comment:
Change this to `Kubecli` to unify naming.

Reply:
kclient is used at many places such as
```go
type BackupManager struct {
	kclient kubernetes.Interface

	clusterName   string
	namespace     string
	etcdTLSConfig *tls.Config

	be backend.Backend
}
```

should we unify the name in a separate pr?

Reply:
> should we unify the name in a separate pr?

Yes.